frontend/fingertips-frontend/app/chart/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { LineChart } from '@/components/pages/chart';
+import {connection} from "next/server";
+
+async function fetchLineChartData() {
+  // We don't want to render this page statically
+  await connection();
+  
+  const response = await fetch('http://localhost:5144/weatherforecast', {
```

Comment:
This needs to use `process.env.FINGERTIPS_API_URL`, as is done in `frontend/fingertips-frontend/app/page.tsx`. This is probably a good point to extract a little utility function to the `lib` directory (please not in `utils.js`, I want to avoid having a single general-purpose "utilities" file!).